libs/sdk-ui-kit/src/Dialog/StylingEditorDialog/StylingEditorDialog.tsx

```diff
@@ -1,16 +1,18 @@
 // (C) 2022 GoodData Corporation
 import React, { useMemo, useState } from "react";
-import { Dialog } from "../Dialog";
-import { Typography } from "../../Typography";
-import cx from "classnames";
-import { StylingExample } from "./StylingExample";
 import { IColorPalette, ITheme, ObjRef } from "@gooddata/sdk-model";
-import { BubbleHeaderSeparator } from "./BubbleHeaderSeparator";
-import { StylingEditorDialogFooter, IStylingEditorDialogFooterProps } from "./StylingEditorDialogFooter";
 import { IntlWrapper } from "@gooddata/sdk-ui";
 import { useIntl } from "react-intl";
```

Comment:
Please move lines 5 ~ 7 up so that import statements of 3rd-party libraries come before import statements of SDK packages.